dev/devicelab/manifest.yaml

```diff
@@ -899,16 +935,6 @@ tasks:
   #   required_agent_capabilities: ["linux/android"]
   #   flaky: true
 
-  # TODO(cyanglaz): Enable this test when we know how to test gold in device labs
```

Comment:
removed commented out code

Reply:
@cyanglaz Do you want to create an issue to cover recreating this test in LUCI?

Reply:
Sure. Done